src/hardware/serialport/softmodem.h

```diff
@@ -91,7 +97,8 @@ class CFifo {
 			static Bits lcount=0;
 			if (lcount<1000) {
 				lcount++;
-				LOG_MSG("MODEM: FIFO Overflow! (adds len %u)",_len);
+				LOG_MSG("MODEM: FIFO Overflow! (adds len %u)",
+				        static_cast<uint32_t>(_len));
```

Comment:
(just a comment)
`%u` for `uint32_t` works on all architectures we build now (and likely will work for almost all in the future). As for now, this code is fine - I'm not sure if this line is even reachable, but that calls for longer and more thorough investigation.

Reply:
Good news: the PRIuPTR warning is solved on 64bit windows gcc with a small define block in types.h.  With time, I suspect we will be able to remove it when the windows libraries understand %llu.